clients/admin-ui/src/types/dictionary-api/models/DataUseDeclaration.ts

```diff
@@ -42,9 +42,6 @@ export type DataUseDeclaration = {
    * The features of processing personal data.
    */
   features?: Array<string>;
-  /**
-   * Whether the legal basis for processing is 'flexible' (i.e. can be overridden in a privacy notice) for this declaration.
```

Comment:
again, another change that came up with my dictionary generate command, figured i'd commit it.

Reply:
Seems like you're missing some docstrings about this in the API model upstream 😉 

All good, don't worry about that in this PR